riotfile.py

```diff
@@ -2976,6 +2976,17 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
                 "google-ai-generativelanguage": [latest],
             },
         ),
+        Venv(
+            name="google_adk",
+            command="pytest {cmdargs} tests/contrib/google_adk",
+            pys=select_pys(min_version="3.9", max_version="3.13"),
```

Comment:
```suggestion
            pys=select_pys(min_version="3.9"),
```

explicitly not supported on 3.14?

Reply:
same, not supported for 3.8?